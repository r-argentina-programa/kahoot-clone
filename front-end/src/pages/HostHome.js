import React from 'react';
import Button from 'react-bootstrap/Button';
import { Link } from 'react-router-dom';
import Container from 'react-bootstrap/Container';
import '../styles/HostHome.css';

const HostHome = () => {
  return (
    <Container className="container centered">
      <Link to="/host/chooseTrivia">
        <Button className="to-trivia-select" variant="info">
          Create/start trivia
        </Button>
      </Link>{' '}
    </Container>
  );
};
export default HostHome;
