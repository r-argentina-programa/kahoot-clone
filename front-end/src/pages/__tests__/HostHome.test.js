import HostHome from '../HostHome';
import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <HostHome />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
