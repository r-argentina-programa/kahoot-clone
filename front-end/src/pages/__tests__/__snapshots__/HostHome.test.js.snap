// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container centered container"
>
  <a
    href="/host/chooseTrivia"
    onClick={[Function]}
  >
    <button
      className="to-trivia-select btn btn-info"
      disabled={false}
      type="button"
    >
      Create/start trivia
    </button>
  </a>
   
</div>
`;
